MerlinAU.sh

```diff
@@ -291,38 +291,79 @@ _WaitForYESorNO_()
    fi
 }
 
-##-------------------------------------##
-## Added by Martinski W. [2023-Dec-26] ##
-##-------------------------------------##
-LockFilePath="/tmp/var/${ScriptFNameTag}.LOCK"
-LockFileMaxSecs=600  #10-min "hold"#
+##----------------------------------------##
+## Modified by Martinski W. [2024-Nov-14] ##
+##----------------------------------------##
+readonly LockFilePath="/tmp/var/${ScriptFNameTag}.LOCK"
+LockFileMaxAgeSecs=600  #10-minutes#
+LockWaitTimeoutSecs=120
+
 _ReleaseLock_() { rm -f "$LockFilePath" ; }
 
+if [ $# -eq 0 ] || [ -z "$1" ]
+then  #Interactive Mode#
+    LockFileMaxAgeSecs=1200
+    LockWaitTimeoutSecs=10
+elif [ "$1" = "addCronJob" ]
+then  #Special Case#
+    LockFileMaxAgeSecs=1200
+    LockWaitTimeoutSecs=600
+fi
```

Comment:
Ah; I see what you did here; and we can add as many cases as we want for staggered fashion :)